couchable/core.py

```diff
@@ -201,6 +211,11 @@ def __init__(self, url=None, db=None, exists=None, timeout=None):
 
         self._db_pid = None
         self._db = None
+        self._done_dict = collections.OrderedDict()
+        self._cycle_set = set()
+        self._additiveOnly = None
+        self._skip_list = []
+
```

Comment:
Without this initialization some of couchables own test cases were failing, when not executed all in the correct order. Making these a proper initialization, removes the possible error of accessing uninitialized attributes.